lambda/latest_image_tracker/template.yml

```diff
@@ -24,14 +24,17 @@ Resources:
           Properties:
             Topic: !Ref ImageBuilderSNSTopic
       Policies:
-          - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+          # - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
```

Comment:
We shouldn't need to make this change. Was this failing or?

Reply:
I tested this as well and you are correct, this fails as SAM is already adding it. Instead of commenting, lets remove the line.